### ISSUE TITLE
[netperf] Debian bookworm installation requires non-free and non-free-firmware in sources.list

### ISSUE THREAD

**Issue by mpenning** [tohojo/flent#298]
FYI, under debian bookworm, I had to configure `/etc/apt/sources.list` with `non-free*` in order to install `netperf`...

```none
$ cat /etc/apt/sources.list
deb http://deb.debian.org/debian bookworm main non-free non-free-firmware
deb http://deb.debian.org/debian bookworm-updates main
deb http://security.debian.org/debian-security bookworm-security main
deb http://ftp.debian.org/debian bookworm-backports main
```

If we could update the README, that would be awesome... if not, at least this ticket is a pointer.

Thanks for a great network test tool!

**Comment by tohojo** [comment]
Thanks for the note. There's actually a two year old Debian bug to get netperf moved out of non-free, but unfortunately this has not happened :(

https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1003833

**Comment by mpenning** [comment]
@tohojo should we just close this ticket?  The Debian `non-free*` portion was only mentioned because I had to figure out how to install netperf on my system.

**Comment by freysteinn** [comment]
> @tohojo should we just close this ticket? The Debian `non-free*` portion was only mentioned because I had to figure out how to install netperf on my system. BTW, the Debian `iperf` version 2 binary doesn't work from the Debian distro because it does not support the `iperf --enhancedreports` flag.

Actually, iperf2 deprecated the --enhancedreports flag and replaced it with --enhanced in 2.0.14.a. The current Flent version checks for this.

I have made many changes to the iperf2 runner; however, I have yet to submit a pull request because I made some changes that aren't fit for upstream. At some point, I will create a pull request that will update the iperf2 runner with more functionality.

**Comment by Infinoid** [comment]
> Thanks for the note. There's actually a two year old Debian bug to get netperf moved out of non-free, but unfortunately this has not happened :(

Looks like they got it done, it's in Debian Trixie (testing) and sid (unstable).

https://packages.debian.org/search?keywords=netperf shows the newer distros no longer tagged [`non-free`].

**Comment by tohojo** [comment]
Yup, and indeed the latest packaged version of Flent already adds netperf as a "suggest". So I guess we can close this now :)
